src/platform/linux/kmsgrab.cpp

```diff
@@ -164,19 +164,48 @@ namespace platf {
   if (string == x) return DRM_MODE_CONNECTOR_##y
 
       _CONVERT("VGA"sv, VGA);
+      _CONVERT("DVII"sv, DVII);
       _CONVERT("DVI-I"sv, DVII);
+      _CONVERT("DVID"sv, DVID);
       _CONVERT("DVI-D"sv, DVID);
+      _CONVERT("DVIA"sv, DVIA);
       _CONVERT("DVI-A"sv, DVIA);
+      _CONVERT("Composite"sv, Composite);
+      _CONVERT("SVIDEO"sv, SVIDEO);
       _CONVERT("S-Video"sv, SVIDEO);
       _CONVERT("LVDS"sv, LVDS);
+      _CONVERT("Component"sv, Component);
+      _CONVERT("9PinDIN"sv, 9PinDIN);
       _CONVERT("DIN"sv, 9PinDIN);
       _CONVERT("DisplayPort"sv, DisplayPort);
       _CONVERT("DP"sv, DisplayPort);
+      _CONVERT("HDMIA"sv, HDMIA);
       _CONVERT("HDMI-A"sv, HDMIA);
       _CONVERT("HDMI"sv, HDMIA);
+      _CONVERT("HDMIB"sv, HDMIB);
       _CONVERT("HDMI-B"sv, HDMIB);
+      _CONVERT("TV"sv, TV);
       _CONVERT("eDP"sv, eDP);
+      _CONVERT("VIRTUAL"sv, VIRTUAL);
+      _CONVERT("Virtual"sv, VIRTUAL);
       _CONVERT("DSI"sv, DSI);
+      _CONVERT("DPI"sv, DPI);
+      _CONVERT("WRITEBACK"sv, WRITEBACK);
+      _CONVERT("Writeback"sv, WRITEBACK);
+      _CONVERT("SPI"sv, SPI);
+#ifdef DRM_MODE_CONNECTOR_USB
+      _CONVERT("USB"sv, USB);
+#endif
```

Comment:
Do you have a reference where this list comes from?

Reply:
Added a comment in the code linking to the various places these came from.